### DISCUSSION TITLE
Strange BunnyCDN nameserver issue

### DISCUSSION THREAD

**Discussion by szepeviktor** [oerdnj/deb.sury.org#1903] [General]
```bash
echo @ns{1..3}.bunnydns.com. | xargs -t -n1 -- dig +short debsuryorg.b-cdn.net. AAAA
```
```
dig +short debsuryorg.b-cdn.net. AAAA @ns1.bunnydns.com.
2a01:4f9:2b:2f5e::2
dig +short debsuryorg.b-cdn.net. AAAA @ns2.bunnydns.com.
dig +short debsuryorg.b-cdn.net. AAAA @ns3.bunnydns.com.
2a01:4f9:2b:2f5e::2
```

These 3 nameservers do not give consistent answers.

**Comment by szepeviktor** [comment]
Even IPv4 addresses as different.
What could be going on?

**Comment by szhu25** [comment]
Because Bunny.net (Bunny CDN) does not support IPv6 in all locations. If you happen to reach a location / zone where Bunny does not have IPv6... then it just won't return it. 

> In regards to IPV6, we are gradually and slowly rolling it out across all of our services. 
    - Bunny Engineer's reply on 2022-12-07

I don't know how or why, but BunnyCDN will return different responses (IPv4/IPv6) and it's "normal" aka ordinary for Bunny... 
Different IPv4 address would be fine, as long as they still return IPv4 addresses I guess (Unlike how they are treating their IPv6 response...)?
